packages/amesos2/test/adapters/Tpetra_CrsMatrix_Adapter_UnitTests.cpp

```diff
@@ -202,7 +199,7 @@ namespace {
     }
     eye->fillComplete();
     // Create using non-member function
-    RCP<ADAPT> adapter  = Amesos2::createMatrixAdapter<MAT>(eye);
```

Comment:
@iyamazaki does commenting this portion of code remove a portion of the test, or was this unused?

Reply:
@ndellingwood, I don't think this will remove the portion of the test, and the variable is just unused?